SW/src/svile/sw/SWarena.php

```diff
@@ -521,12 +521,15 @@ private function start()
                 if ($p->getAttributeMap() != null) {//just to be really sure
                     $p->setHealth($this->pg->configs['join.health']);
                     $p->setFood(20);
+                    $p->setGamemode(Player::SURVIVAL);
                 }
                 $p->sendMessage($this->pg->lang['game.start']);
                 if ($p->getLevel()->getBlock($p->floor()->subtract(0, 2))->getId() == 20)
                     $p->getLevel()->setBlock($p->floor()->subtract(0, 2), Block::get(0), true, false);
+
                 if ($p->getLevel()->getBlock($p->floor()->subtract(0, 1))->getId() == 20)
                     $p->getLevel()->setBlock($p->floor()->subtract(0, 1), Block::get(0), true, false);
+
```

Comment:
and I dont think I added those, I only add when game has started that set player back to be able to break blocks aka survival mode